proto/streamapis/model/v1/model.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
```

Comment:
Is it a copy of [this file](https://github.com/proxima-one/stream-db/blob/master/api/streamapis/model/v1/model.proto)? How do we plan to synchronize it? I would suggest to come up with a way to store .proto files in a single place and import it somehow for usages in other places. As worst, it can be a git submodule. But I think Buf packages are designed specifically for that purpose.